tutorial.md

```diff
@@ -24,41 +24,41 @@ and can be executed by uploading the text file [4Q2006.txt](./sample-data/revenu
 
 Under **Extractors**, drag and drop **Input Documents** on the canvas. Configure with document [4Q2006.txt](./sample-data/revenue by division/financial statements/4Q2006.txt). Click **Upload**, then **Close**.
```

Comment:
This wasn't introduced by this PR but this link isn't working for me - I think it might be because of the spaces in the linked path?